spec/spec_helper.rb

```diff
@@ -41,6 +41,8 @@
     stub_geo_ip
   end
 
+  ENV['RAILS_HOST_NAME'] = 'www.example.com'
```

Comment:
you could also use cryptopus.example.com ...

Reply:
No it has to be the rspec specific hostname